content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Pause and Restore Palette Virtual Clusters"
+metaTitle: "Palette Dev Engine Pause and Restore Palette Virtual Clusters"
+metaDescription: "Pause and Restore Palette Virtual Clusters"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+
+# Overview
+
+Palette allows the pause and resume of Palette Virtual Clusters when not in use. This feature enables the users to optimize resource utilization by pausing the Virtual Clusters, not in use. This adds significant flexibility in managing operating costs and resource management for the Palette Virtual Clusters. 
+
+# Prerequisite
+
+* Running [Palette Virtual Cluster](/devx/sandbox-clusters#createyoursandboxcluster:).
+
+#  How to Pause and Resume your Palette Virtual Cluster
+
+Invoke the pause and resume operations from the Palette Console.
+
+* Log in to the **Palette Dev Engine** Console.
+
+
+* Select Palette Virtual Cluster from the **left ribbon menu**.
+
+
+* Go to the cluster details page by clicking the name of the Palette Virtual Cluster to be paused.
+
+
+* Click **Settings** and Select the **Pause** option. To resume a paused Virtual Cluster, select the **Resume** option.
```

Comment:
Because these are sequential steps, we should use an ordered list with numbers instead of generic bullet points.